server/save.py

```diff
@@ -223,6 +223,17 @@ def upgrade(version):
             )
         db.foreign_keys = True
         Constants.update(save_version=Constants.save_version + 1).execute()
+    elif version == 16:
+        from peewee import TextField
+        migrator = SqliteMigrator(db)
+        db.foreign_keys = False
+        with db.atomic():
+            migrate(
+                migrator.add_column("location_user_option", "unit_size_unit",
```

Comment:
```suggestion
                migrator.add_column("location", "unit_size_unit",
```
the LocationUserOption table is another table than the Location table.  The former are some client specific settings like the current view position in the location, whereas the latter has the concrete settings that apply to everyone.